demo/src/lib.rs

```diff
@@ -9,15 +9,29 @@ pub struct MyApp {
     geoportal_tiles: Tiles,
     map_memory: MapMemory,
     satellite: bool,
+    texture: Texture,
+    rotate: f32,
+    x_scale: f32,
+    y_scale: f32,
```

Comment:
Could you group them into some struct, to point out that they are related to the images drawing part of the demo?